README.md

```diff
@@ -1,19 +1,32 @@
 # Caiquenator
 
-To start your Phoenix server:
+On a fateful day, looking for developers to review your PR, our hero
+@caiquemitsuoka created a way to engage more people to do that for us.
+And we, recognizing a good idea and thinking about all the other developers who
+certainly have the same difficulty, decided to create Caiquenator! 🎆
 
-  * Install dependencies with `mix deps.get`
-  * Install Node.js dependencies with `cd assets && npm install`
-  * Start Phoenix endpoint with `mix phx.server`
+Check our wonder at [caiquenator.herokuapp.com](https://caiquenator.herokuapp.com)
+
+## Setup
+
+- Install dependencies with `mix deps.get`
+- Install Node.js dependencies with `cd assets && npm install`
```

Comment:
Here you ask to user to go to assets folder (`cd assets`) but hasn't mention to go back to app root folder and execute `mix test` bellow. 

WDYT about change the line 14 to something like this: `On application root folder, run tests with 'mix test'` ?

Reply:
f0d925b0e7662f5926695e357e5d41542080913b